packages/blockchain/contracts/Event.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+contract Event {
+    address public owner;
+    string public title;
+    string public description;
+    string public location;
+    string public eventType;
+    string public image;
+    string public eventID;
+    uint public date;
+
+    mapping(address => bool) public isAttending;
+    address[] private attendeesArray;
+
+    event AttendeeAdded(address attendee);
+    event AttendeeRemoved(address attendee);
+
+    constructor(
+        string memory _title,
+        string memory _description,
+        string memory _location,
+        string memory _eventType,
+        string memory _image,
+        uint _date,
+        string memory _eventID
+    ) {
+        owner = tx.origin;
+        title = _title;
+        description = _description;
+        location = _location;
+        eventType = _eventType;
+        image = _image;
+        date = _date;
```

Comment:
date could be a range as well for some events. eg: `from 23st sept - to 24th sept` 
can we try timestamps ? there could be events that 1 -2 hrs long

Reply:
Yes!! Timestamps are numbers so will be ok if we add them here! If you check above you'll see the date is a `uint` value